File: Servers/Python/Aiohttp/routers/json_response.py
from aiohttp import web

from Servers.Python.utils.path import JsonResponse
import Servers.test_data

router = web.RouteTableDef()


@router.get(JsonResponse.one_k_json)
async def one_k_json(_: web.Request) -> web.Response:
    return web.json_response(Servers.test_data.JSON_1K)


@router.get(JsonResponse.ten_k_json)
async def ten_k_json(_: web.Request) -> web.Response:
    return web.json_response(Servers.test_data.JSON_10K)


@router.get(JsonResponse.hundred_k_json)
async def hundred_k_json(_: web.Request) -> web.Response:
    return web.json_response(Servers.test_data.JSON_100K)


@router.get(JsonResponse.one_kk_json)
async def one_kk_json(_: web.Request) -> web.Response:
    return web.json_response(Servers.test_data.JSON_1M)


@router.get(JsonResponse.five_kk_json)
async def five_kk_json(_: web.Request) -> web.Response:
    return web.json_response(Servers.test_data.JSON_5M)

File: Servers/JS/Koa/routers/jsonResponse.js
const oneKJsonData = require("../../../test_data/1K.json");
const tenKJsonData = require("../../../test_data/10K.json");
const hundredJsonData = require("../../../test_data/100K.json");
const oneKKJsonData = require("../../../test_data/1M.json");
const { jsonResponse } = require("../../utils/paths");

const fiveKKJsonData = require("../../../test_data/5M.json");

const Router = require("koa-router");
const router = new Router();

router.get(jsonResponse.oneKJson, async (ctx) => {
  ctx.body = oneKJsonData;
});
router.get(jsonResponse.tenKJson, async (ctx) => {
  ctx.body = tenKJsonData;
});
router.get(jsonResponse.hundredKJson, async (ctx) => {
  ctx.body = hundredJsonData;
});
router.get(jsonResponse.oneKKJson, async (ctx) => {
  ctx.body = oneKKJsonData;
});
router.get(jsonResponse.fiveKKJson, async (ctx) => {
  ctx.body = fiveKKJsonData;
});

module.exports = router;
